README.md

```diff
@@ -28,13 +27,298 @@ pip install -U pyinstaller-hooks-contrib
 
 Either `a-package` works fine without a hook, or no-one has contributed hooks.
 If you'd like to add a hook, or view information about hooks,
-please see [the wiki](https://github.com/pyinstaller/pyinstaller-hooks-contrib/wiki).
+please see below.
 
 
 ## I want to help!
 
-Please start by providing pull requests and helping solve issues.
+If you've got a hook you want to share then great!
+The rest of this page will walk you through the process of contributing a hook.
+If you've been here before then you may want to skip to the [summary checklist](#summary)
+
+**Please provide one hook per pull request! If you have more than one then submit them in separate pull requests.**
```

Comment:
See normally I'd still rather they come as separate PRs which just cross reference each other. It's mostly just so that we can squash merge each one and therefore is doesn't matter if the PR author makes a mess of git history.